daemon/daemon_unix.go

```diff
@@ -455,12 +455,19 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 		ipamV4Conf.AuxAddresses["DefaultGatewayIPv4"] = config.Bridge.DefaultGatewayIPv4.String()
 	}
 
-	var ipamV6Conf *libnetwork.IpamConf
+	var (
+		ipamV6Conf     *libnetwork.IpamConf
+		deferIPv6Alloc bool
+	)
 	if config.Bridge.FixedCIDRv6 != "" {
 		_, fCIDRv6, err := net.ParseCIDR(config.Bridge.FixedCIDRv6)
 		if err != nil {
 			return err
 		}
+
+		ones, _ := fCIDRv6.Mask.Size()
+		deferIPv6Alloc = ones <= 80
```

Comment:
Thanks for the background, I think it's worth a code comment (or people reading this will definitely scratch their heads).

I don't really understand why you would need a variable if you just called `libnetwork.NetworkOptionDeferIPv6Alloc` at that point of the code, but I'll trust you with that :-/